src/popup/plugins/veeValidate.ts

```diff
@@ -112,6 +116,16 @@ defineRule(
   (value: string, [arg]: [number]) => (value && value.length <= arg) || tg('validation.maxLength', [arg]),
 );
 
+defineRule(
+  'password_min_len',
+  (value: string, [arg]: [number]) => (value && value.length >= arg) || tg('validation.passwordMinLength', [arg]),
+);
```

Comment:
Vee validate has it's default rule for min/max length. I suggest to use the generic one. The error could also be generic in this case - to not use word "password". This way we could reuse this in some other cases.

Reply:
@onvisions What do you think about creating/using a reusable validation for minimum length of values?

Reply:
> @onvisions What do you think about creating/using a reusable validation for minimum length of values?

I don't see the point from user perspective if you have already developed aproperly working solution with more accurate custom wording and validation messages etc. From code perspective I can't really say. Is it really better to use generic validation when we already have validation logic and not only for the password but also for all input fields in the wallet? 
IMO we should keep it as designed and already implemented. However I'd like to hear the opinion of the other developers and after all @smaroudas will have the final word.

Reply:
The rule is simple: less code, less stuff that can break and less places to maintain. Writing code only for one specific case is in most cases bad idea. But I'm not forcing anything here as this is a tiny thing.